daemons/ipa-kdb/ipa_kdb_kdcpolicy.c

```diff
@@ -108,7 +110,8 @@ ipa_kdcpolicy_check_as(krb5_context context, krb5_kdcpolicy_moddata moddata,
      * apply them */
     if (pol_limits != NULL) {
         if (pol_limits->max_life != 0) {
-            *lifetime_out = pol_limits->max_life;
+            /* Set lifetime_out to a random value within the last hour of lifetime */
+            *lifetime_out = (pol_limits->max_life - 3600) + (rand() % pol_limits->max_life);
```

Comment:
`rand()` should be taken modulo 3600.
Also please protect against too small values. For example, we can skip applying jitter for `max_life` smaller than 3600 seconds.